code/modules/reagents/machinery/chem_master.dm

```diff
@@ -582,7 +592,18 @@ var/global/list/pillIcon2Name = list("oblong purple-pink", "oblong green-white",
 			//
 			// PILL ICONS
 			//
-			dat += generate_pill_icon_div()
+
+			dat += "<a href='?src=\ref[src];pill_icon_toggle=1'>Toggle Additional Pill Icons</a>"
```

Comment:
This button will show even if no beaker is added if you just put it into the `generate_pill_icon_div()` proc like I said